certora/specs/SocialRecoveryModule.spec

```diff
@@ -1,3 +1,27 @@
-rule demo() {
-    assert true;
+using SafeHarness as safeContract;
+
+methods {
+    // Safe Functions
+    function safeContract.isModuleEnabled(address) external returns (bool) envfree;
+    function safeContract.disableModule(address, address) external;
```

Comment:
I _think_ this does not need to be declared (since functions are `envfree` by default).

Reply:
Yes, that is right 👍🏾